### PR TITLE
Remove unwanted readme heading

### PR DESCRIPTION
**Pull request by timabell**
I understand this was introduced by mistake.

Cleanup (and a little PR demo for Nick :smile: )

### CODE REVIEW

**Review by kicferk1 — COMMENTED** [review]
I very much doubt it was a mistake. It stands for `Zero to Hero`, and could perhaps be changed to `Setup instructions` to make it clearer.

Otherwise we have header jump, from H1 to H3 in our markdown :P

**Comment by ethanmills** [comment]
Unwanted by you, perhaps

**Comment by timabell** [comment]
> Unwanted by you, perhaps

I clearly don't understand what this is for, I still thought this was an editor mistake.

**Comment by timabell** [comment]
So, it's actually an acronym that's intended to mean "how to set up this project quickly"? I totally didn't get that, and I don't think that kind of obscure reference is appropriate for a readme on a public gov repo where many people will need to understand the nature of the project. I suggest someone changes it to be in plain english.

**Comment by timabell** [comment]
It's annoying that I can't change the branch name without breaking the PR in github, I think I'll open a new PR with a new branch name
